src/main/java/recipeio/Constants.java

```diff
@@ -28,5 +28,17 @@ public class Constants {
     public static final String FIND_BY_KEYWORD = "kw";
     public static final String FIND_BY_DATE = "date";
     public static final String FIND_BY_ALLERGY = "allergy";
+    public static final String FIND_BY_MEAL = "meal";
 
+    public static final String MEAL_CAT_GENERAL = "general";
+    public static final String MEAL_CAT_DINNER = "dinner";
+    public static final String MEAL_CAT_BREAKFAST = "breakfast";
+    public static final String MEAL_CAT_LUNCH = "lunch";
+    public static final String MEAL_CAT_APPETIZER = "appetizer";
+    public static final String MEAL_CAT_DESSERT = "dessert";
+
+    //Test find commands
+    public static final String COMMAND_INVALID_MEAL_CAT = "find meal fdjsjfsad";
+    public static final String COMMAND_EXIST_MEAL = "find meal breakfast";
+    public static final String COMMAND_NOT_EXIST_MEAL = "find meal appetizer";
```

Comment:
Let's not put these constants here, since they are used in the test file only.

You could move these lines directly into the test file.